project/Dependencies.scala

```diff
@@ -232,7 +238,7 @@ object Dependencies {
       betterMonadicFor
     ) ++ testKit ++ quill ++ silencer ++ monocle
 
-    lazy val dexLoad: Seq[ModuleID] = Seq(diffx) ++ silencer
+    lazy val dexLoad: Seq[ModuleID] = Seq(diffx) ++ silencer ++ load
```

Comment:
..use it here like that: ` lazy val dexLoad: Seq[ModuleID] = Seq(diffx, pureConfig) ++ silencer`?

Reply:
done